tests/test_deltas.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import torch
```

Comment:
I also think it'd be nicer if we can completely get away from using torch/torchaudio. The equations at https://pytorch.org/audio/functional.html#compute-deltas is broken now, but..
i)  maybe after waiting for them to fix it, then implement the test on numpy according to the equation
ii) or, use https://librosa.github.io/librosa/generated/librosa.feature.delta.html (not 100% sure if they refer to the same operation but i believe they should)?

Reply:
Done